Jenkinsfile.cloud

```diff
@@ -50,17 +50,21 @@ node(NODE) {
 
     // Check if there's a new version out.
     // Really, we should be checksumming e.g. the ks and tdl too.
-    def latest_commit, last_build_commit, latest_version, force_nocache
+    def latest_commit, last_build_meta, latest_version, force_nocache
     stage("Check for Changes") {
         latest_commit = utils.sh_capture("ostree --repo=repo rev-parse ${ref}")
         latest_version = utils.get_rev_version("repo", "${latest_commit}")
         if (fileExists('force-nocache-build')) {
             force_nocache = readFile('force-nocache-build').trim();
         }
-        last_build_commit = utils.sh_capture("cat ${images}/cloud/latest/commit.txt || :")
+        def previous_meta_path = "${images}/cloud/latest/meta.json";
+        if (fileExists(previous_meta_path)) {
+            last_build_meta = readJSON file: previous_meta_path;
+        }
     }
 
-    if (!params.DRY_RUN && (latest_commit == last_build_commit && latest_commit != force_nocache)) {
+    def previously_built_commit = (last_build_meta != null && latest_commit == last_build_meta["commit"]);
```

Comment:
Shouldn't this be last_build_meta["ostree-commit"]

Reply:
Yep good catch!  Fixed :arrow_down: